variables.tf

```diff
@@ -530,6 +530,36 @@ variable "sku_tier" {
   default     = "Free"
 }
 
+variable "storage_profile_blob_driver_enabled" {
+  type        = bool
+  description = "(Optional) Is the Blob CSI driver enabled? Defaults to false"
+  default     = null
+}
+
+variable "storage_profile_disk_driver_enabled" {
+  type        = bool
+  description = "(Optional) Is the Disk CSI driver enabled? Defaults to true"
+  default     = null
+}
+
+variable "storage_profile_disk_driver_version" {
+  type        = string
+  description = "(Optional) Disk CSI Driver version to be used. Possible values are v1 and v2. Defaults to v1."
+  default     = null
+}
+
+variable "storage_profile_file_driver_enabled" {
+  type        = bool
+  description = "(Optional) Is the File CSI driver enabled? Defaults to true"
+  default     = null
```

Comment:
According to the [schema](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/internal/services/containers/kubernetes_cluster_resource.go#L1083-L1087), the `default` should be `true`.

Reply:
Changed